starfish/experiment/__init__.py

```diff
@@ -125,6 +134,23 @@ def __init__(
         self._extras = extras
         self._src_doc = src_doc
 
+    def __repr__(self):
+
+        # truncate the list of fields of view if it is very long
+        if len(self._fovs) > 4:
+            fovs = self._fovs[:4] + ['...,\n']
+        else:
+            fovs = self._fovs
+
+        # convert fovs to string, careful not to re-convert the ellipsis
+        string_fovs = ',\n'.join(repr(f) if not isinstance(f, str) else f for f in fovs)
+
+        # return the formatted string
+        return (
+            f"<starfish.Experiment (FOVs={len(self._fovs)})>\n"
+            f"FOVs:\n[{string_fovs}]"
```

Comment:
square brackets are kind of weird.  i would almost prefer a numbered list.

Reply:
Square brackets say "I'm a `List`". Curly brackets with names say "I'm a `Dict`". I want this class to both communicate and act like one of those things. Because the `fov.name` were `None`, and it generates a list of FOVs, I made this more list-like. I'm open to both (`Dict` might be better!).